contracts/test/veLON/Withdraw.t.sol

```diff
@@ -35,10 +42,14 @@ contract TestVeLONWithfraw is TestVeLON {
 
         // pretend 1 week has passed and the lock not expired
         vm.warp(block.timestamp + 1 weeks);
+
+        // calculate the panalty
+        uint256 expectedPanalty = ((DEFAULT_STAKE_AMOUNT.mul(veLon.earlyWithdrawPenaltyRate())).div(veLon.PENALTY_RATE_PRECISION()));
+        uint256 expectedAmount = DEFAULT_STAKE_AMOUNT.sub(expectedPanalty);
+        vm.expectEmit(true, true, true, true);
+        emit Withdraw(user, false, tokenId, expectedAmount, expectedPanalty, block.timestamp);
         vm.prank(user);
         veLon.withdrawEarly(tokenId);
-        uint256 balanceChange = DEFAULT_STAKE_AMOUNT.mul(earlyWithdrawPenaltyRate).div(PENALTY_RATE_PRECISION);
-        stakerLon.assertChange(int256(DEFAULT_STAKE_AMOUNT.sub(balanceChange)));
```

Comment:
This assertion should not be remove. Also, `lockedLon` should be checked as well.

Reply:
I originally thinked that the balance change has tested in `expectEmit withdraw` , so  I removed these two line. But I can add them back.

Reply:
Considering them as the same is an assumption. Let's verify it as much as you can.